tests/k6/src/test-delete-file.ts

```diff
@@ -0,0 +1,49 @@
+import {sleep, check} from 'k6';
+import {Options} from "k6/options";
+import {defaults, api} from "./lib";
+
+const files = {
+    'kb_50.jpg': open('./_files/kb_50.jpg', 'b'),
+}
+
+export let options: Options = {
+    insecureSkipTLSVerify: true,
+    iterations: 100,
+    vus: 100,
+};
+
+export const setup = (): void => {
+    console.log("setup for delete")
+    for (let vu=1; vu <= options.vus; vu++) {
+      for (let iter=0; iter < options.iterations/options.vus; iter++) {
+        const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `deletefile_${vu}_${iter}.jpg`)
+        check(res, {
+          'status is 201': () => res.status === 201,
+        })
+      }
+    }
+    console.log("uploaded test files")
+    sleep(1)
```

Comment:
the sleep isn't required here. the setup code runs once within a test for all VUS, or do i miss something?